testnets/bitcanna-dev-6/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "bitcanna-dev-6",
```

Comment:
please change to 'bitcannatestnet', and update the folder name to this as well. dashes/hyphens are not allowed.

Reply:
changed 👍